.travis.yml

```diff
@@ -1,4 +1,10 @@
 language: ruby
 rvm:
  - 2.3.3
-script: bundle exec rake jasmine:ci
+script:
+  - bundle exec rake jasmine:ci
+  - gem install middleman
```

Comment:
Good idea, I've pulled it into a shell script. Battling Travis at the moment, maybe that fixes it.